src/fdk_rdf_parser/parse_functions/qualified_attribution.py

```diff
@@ -0,0 +1,34 @@
+from typing import List, Optional
+
+from rdflib import Graph, URIRef
+
+from fdk_rdf_parser.rdf_utils import dcatURI, provURI, objectValue
+
+from fdk_rdf_parser.classes import QualifiedAttribution, PublisherId
+from fdk_rdf_parser.rdf_utils import objectValue
+
+
+def extractQualifiedAttributions(
+    graph: Graph, subject: URIRef, predicate: URIRef
+) -> Optional[List[PublisherId]]:
+    values = []
+    for resource in resourceList(graph, subject, predicate):
+        publisherUri = None
+        publisherId = None
+        role = objectValue(graph, resource, dcatURI("hadRole"))
+
+        if isinstance(resource, URIRef):
+            publisherUri = objectValue(graph, resource, provURI("agent"))
+            publisherId = publisherUri.split("/").pop()
```

Comment:
trekk dette ut i utils elns, tror vi har behov for det flere plasser

Reply:
Og gjør det mer robust, i tilfelle det er link til noe annet enn enhetsregisteret

Reply:
kun godta 9 tall?
Og ha med en test for når den ikke finner orgnr

Reply:
Done